docs/Guide.Mocking.md

```diff
@@ -57,3 +57,26 @@ Whenever Metro runs with `RN_SRC_EXT` environment variable set, it will override
 > RN_SRC_EXT=e2e.js ./gradlew assembleRelease
 ``` 
 
+#### Example of how to mock a module
+If you want to mock a module, here is an example of how to do it: 
+1. Follow the steps above on section [Configuration](#Configuration)
```

Comment:
Should be "steps above in the [Configuration](#Configuration) section"

Reply:
Sweet, thank you for the feedback! 🎉